xilem/examples/variable_clock.rs

```diff
@@ -0,0 +1,221 @@
+        let date_time_in_self = data.now_utc.to_offset(self.offset);
+        sized_box(flex((
+            flex((
+                prose(self.region),
+                FlexSpacer::Flex(1.),
+                label(format!("UTC{}", self.offset)).brush(
+                    if data.local_offset.is_ok_and(|it| it == self.offset) {
+                        Color::ORANGE
+                    } else {
+                        masonry::theme::TEXT_COLOR
+                    },
+                ),
+            ))
+            .must_fill_major_axis(true)
+            .direction(Axis::Horizontal)
+            .flex(1.),
+            flex((
+                variable_label(
+                    date_time_in_self
+                        .format(format_description!("[hour repr:24]:[minute]:[second]"))
+                        .unwrap()
+                        .to_string(),
+                )
+                .text_size(48.)
+                // Use the roboto flex we have just loaded.
+                .with_font(FontStack::List(&[FontFamily::Named("Roboto Flex")]))
+                .target_weight(data.weight, 400.),
+                FlexSpacer::Flex(1.0),
+                (data.local_now().date() != date_time_in_self.date()).then(|| {
+                    label(
+                        date_time_in_self
+                            .format(format_description!("([day] [month repr:short])"))
+                            .unwrap(),
+                    )
+                }),
+            ))
+            .direction(Axis::Horizontal),
+        )))
+        .expand_width()
+        .height(72.)
+    }
+}
+
+fn app_logic(data: &mut Clocks) -> impl WidgetView<Clocks> {
+    flex((
+        // HACK: We add a spacer at the top for Android. See https://github.com/rust-windowing/winit/issues/2308
+        FlexSpacer::Fixed(40.),
+        controls(),
+        TIMEZONES.iter().map(|it| it.view(data)).collect::<Vec<_>>(),
+    ))
+}
+
+fn controls() -> impl WidgetView<Clocks> {
+    flex((
+        button("Increase", |data: &mut Clocks| {
+            data.weight = (data.weight + 100.).clamp(1., 1000.);
+        }),
+        button("Decrease", |data: &mut Clocks| {
+            data.weight = (data.weight - 100.).clamp(1., 1000.);
+        }),
+        button("Minimum", |data: &mut Clocks| {
+            data.weight = 1.;
+        }),
+        button("Maximum", |data: &mut Clocks| {
+            data.weight = 1000.;
+        }),
+        button("Refresh Offset", |data: &mut Clocks| {
+            // This does nothing because of `time`'s soundness checks...
+            data.local_offset = UtcOffset::current_local_offset().or(data.local_offset);
+        }),
+    ))
+    .direction(Axis::Horizontal)
+}
+
+/// A subset of [Roboto Flex](https://fonts.google.com/specimen/Roboto+Flex), used under the OFL.
+/// This is a variable font, and so can be.
+/// The version in the repository supports the numbers 0-9 and `:`, as it is included for this example,
```

Comment:
Nice, I hope this avoids further misunderstandings (and good that there's no unsupported letter directly in the example, as I think that just produces avoidable discussions)